src/Writers/HTMLWriter.jl

```diff
@@ -166,17 +165,23 @@ function render_head(ctx, navnode, additional_scripts)
     @tags head meta link script title
     src = get(navnode.page)
     page_title = "$(mdflatten(pagetitle(ctx, navnode))) · $(ctx.doc.user.sitename) documentation"
+    css_links = [
+        "https://cdnjs.cloudflare.com/ajax/libs/normalize/4.2.0/normalize.min.css",
+        "https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/default.min.css",
+        "https://fonts.googleapis.com/css?family=Lato|Ubuntu+Mono",
+        "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css",
+        relhref(src, ctx.documenter_css),
+    ]
```

Comment:
How do you feel about having external etc. somewhere at the top of the file, instead of being hidden away in the code? I feel that it would be nice to have them in once place, which is also the reason I already have the `const requirejs_cdn` there.